src/tests/ftest/pool/rebuild_tests.py

```diff
@@ -45,8 +45,8 @@ def run_rebuild_test(self, pool_quantity):
         pools = []
         containers = []
         for index in range(pool_quantity):
-            pools.append(TestPool(self.context, self.log,
-                                  dmg_command=self.get_dmg_command()))
+            pools.append(TestPool(self.context, self.get_dmg_command(),
+                                  log=self.log))
```

Comment:
The `log` argument is defunct for TestPool.create():
```suggestion
            pools.append(TestPool(self.context, self.get_dmg_command()))
```